code/Processing_CloudRegions.py

```diff
@@ -1,44 +1,61 @@
 import argparse
 import Querying_Database as qdb
+import geopandas as gpd
+
+import pandas as pd
 
 from collections import defaultdict
 from geopy.distance import geodesic
 from math import inf
 from shapely import wkt, shortest_line, LineString, Point
+from sqlite3 import Row as sqlite_Row
+
 
 def find_closest_paths(lat: float, lon: float, db_path: str, max_distance: float):
     querier = qdb.queryDatabase(db_path)
     query = f"""SELECT * FROM standard_paths;"""
-    results = querier.execute_query(query)
+    results = querier.execute_query(query, row_factory=sqlite_Row)
     
     paths = []
-    for row in results:
-        fc = row[0]
-        fs = row[1]
-        fcc = row[2]
-        tc = row[3]
-        ts = row[4]
-        tcc = row[5]
-        dist_km = float(row[6])
-        path_wkt = row[7]
-        edge = ((fc, fs, fcc), (tc, ts, tcc))
-
-
-        linestring = wkt.loads(path_wkt)
-        # The linestring is an existing edge in the graph, and the point is the
-        # location of the cloud region. We get the shortest line between these
-        # two geometries, which gives us the shortest distance from a cloud
-        # region to an existing edge in the graph.
-        ps = shortest_line(Point(lon, lat), linestring).coords
-        # Since the WKT format is (lon, lat) and geopy.geodesic takes
-        # (lat, lon) pairs, we need to switch these.
-        p1 = (ps[0][1], ps[0][0])
-        p2 = (ps[1][1], ps[1][0])
-        distance = geodesic(p1, p2)
-        if distance.km < max_distance:
-            paths.append(row)
-
-    return paths
+
+    df = pd.DataFrame(results, columns=results[0].keys())
+
+    gs = gpd.GeoSeries.from_wkt(df['path_wkt'])
+    geodf = gpd.GeoDataFrame(df, geometry=gs, crs="EPSG:4326")
```

Comment:
Adding some comments on what these crs strings mean would be super helpful.

Reply:
Done